package.json

```diff
@@ -10,8 +10,10 @@
     "astro": "astro"
```

Comment:
@schabibi1 shall we include a script for generating types easily via `npm`, and even we can include a pre-commit hook to run it before commiting, so we can see if something has been changed in Storyblok while working async.

Do you like the idea? I can help doing it, if so 💜

Reply:
@Dawntraoz That's practical and I learned that today from the website repo I was working with website team🙌
I included a quick one and tested, it works to pull components and update with regenerating types from npm script✅

I'll push it but feel free to give me the feedback💪

Reply:
It's on #4 😉